### PR TITLE
Zombie infection chance is lowered by armor

### PR DESCRIPTION
**Pull request by SomeguyManperson**
:cl: The CDC
rscadd: Zombie infections are now blocked a % of the time scaling with melee armor
/:cl:

### CODE REVIEW

**Comment by yacabo** [comment]
This is a really good idea, a shame Travis does not agree

**Comment by SomeguyManperson** [comment]
Travis is really mad at everything for some reason I don't understand but I did test it and it works

File: code/modules/zombie/items.dm [hunk 1]
```diff
@@ -33,7 +33,10 @@
 		return
 	else if(isliving(target))
 		if(ishuman(target))
-			try_to_zombie_infect(target)
+			var/mob/living/human/H = target
+			var/flesh_wound = ran_zone(user.zone_selected)
+			if(prob(100-H.getarmor(flesh_wound, "melee"))
```
Comment (alexkar598) [inline code/modules/zombie/items.dm]:
```suggestion
			if(prob(100-H.getarmor(flesh_wound, "melee")))
```

Should make this PR compilable

**Review by alexkar598 — CHANGES_REQUESTED** [review]

File: code/modules/zombie/items.dm [hunk 2]
```diff
@@ -33,7 +33,10 @@
 		return
 	else if(isliving(target))
 		if(ishuman(target))
-			try_to_zombie_infect(target)
+			var/mob/living/human/H = target
```
Comment (alexkar598) [inline code/modules/zombie/items.dm]:
```suggestion
			var/mob/living/carbon/human/H = target
```

**Review by alexkar598 — DISMISSED** [review]
God your a shitty coder

**Comment by Bibby0110** [comment]
Says Alexkar

**Comment by Nickvr628** [comment]
I like this

**Review by Amelia0010 — APPROVED** [review]

**Pull request merged**